app/static/js/manageDepartments.js

```diff
@@ -26,8 +27,87 @@ $(document).ready( function(){
+    $('.supervisorDeptModal').on('click', function() {
+      $('#manageSupervisorContent .modal-header .department-name')
+                .replaceWith(`<h2 class='department-name' id="departmentModalSelect" data-department-id="${this.id}">
+                                    ${$(`#${this.id}`).html()}
+                              </h2>`)
+      getSupervisorsInDepartment(this.id)
+    })
+}
+
+
+$("#supervisorModalSelect").on('change', function() {
+  let supervisorID = $('#supervisorModalSelect :selected').val()
+  let departmentID = $('#departmentModalSelect').data('department-id')
+  addSupervisorToDepartment(supervisorID, departmentID, ()=>getSupervisorsInDepartment(departmentID))
+  
+})
+
+  
+
+function getSupervisorsInDepartment(departmentID) {
+    $.ajax({
+    method: "GET",
+    url: `/admin/manageDepartments/${departmentID}`,
+    success: function(departmentsAndSupervisors) {
+      let currentDepartment=departmentsAndSupervisors[0]
+      let supervisors=departmentsAndSupervisors[1]
+      
+      let supervisorContent = '<div class="changing-content">'
+      for (let i=0; i<supervisors.length; i++) {
+        let supervisorFirstName= supervisors[i]['preferred_name'] ? supervisors[i]['preferred_name'] : supervisors[i]['legal_name']
+        supervisorContent += (`<span class="row">
+                                <div class="col-xs-10">${supervisors[i]['ID']} ${supervisorFirstName} ${supervisors[i]['LAST_NAME']}</div>
+                                <div class='btn btn-danger col-auto removeSupervisorFromDepartment' 
+                                  data-supervisor="${supervisors[i]['ID']}" 
+                                  data-department="${currentDepartment['departmentID']}" 
+                                  id="${supervisors[i]['ID']}-${currentDepartment['departmentID']}">Remove</div>
+                               </span>`)}
+      supervisorContent += ("</div>")
+      $('#manageSupervisorContent .modal-body .changing-content').replaceWith(supervisorContent)
+      
+      $('#manageDepartmentSupervisorModal').modal('show')
+      $('.removeSupervisorFromDepartment').on('click', removeSupervisorFromDepartment)
+    }
+    })
+  }
+
+function removeSupervisorFromDepartment () {
+  let department = $(`#${this.id}`).data('department')
+  let supervisor = $(`#${this.id}`).data('supervisor')
+  let data = {"supervisorID": supervisor, "departmentID": department}
+  $.ajax({
+    method: "POST",
+    url: "/admin/manageDepartments/removeSupervisorFromDepartment",
+    data: data,
+    success: function(response) {
+        if (response == "True") {
+          msgFlash("Supervisor has been removed from department.", 'success')
+          getSupervisorsInDepartment(department)
+        } else {
+          msgFlash("Supervisor is not a member of this department.", "warning")
+        }
+      },
+      error: function() {
+        msgFlash("Failed to remove supervisor, please try again.", "fail")
+      },
+    })
```

Comment:
Watch the indentation here. From this line up to `error: function() {`, we should decrease by one indentation level. The error block should be in line with the success block.